### DISCUSSION TITLE
[버그] 봇 설명란 이모지 입력 위치

### DISCUSSION THREAD

**Discussion by popop098** [koreanbots/koreanbots#470] [제안]
## 재현방법

> 관리하기 페이지에서 봇 설명란부분

## 예상되는 정상적인 동작

> 이모지가 글씨를 가리지 않아야합니다.

## 발생한 문제

> ![image](https://user-images.githubusercontent.com/53284350/130327272-c50b014f-4cdb-4bc7-871f-0d6840873441.png) 이모지가 일부분을 가림

## 빌드 버전

> v2.1.1

## 사양

> OS: 윈도우10, 브라우저: 웨일브라우저

## 확인

- [ x] 중복되는 이슈는 없나요?
- [ x] 해당 버그를 다시 발생시킬 수 있나요?

<!--
마크다운 체크박스 문법입니다.
[x] - 체크
[ ] - 체크하지 않음
으로 작성해주세요.
>

**Comment by wonderlandpark** [comment]
버그가 아닙니다. 제안이므로 Discussion으로 이동합니다.

**Comment by LanaKee** [comment]
macOS 12.0 버전에서는 크롬,사파리 두 브라우저모두 해당현상이 일어나지 않습니다
참고하시길 바랍니다
